live-examples/js-examples/array/array-isarray.js

```diff
@@ -0,0 +1,11 @@
+console.log(Array.isArray([1, 3, 5]));
+// expected output: true
+
+console.log(Array.isArray('[]'));
+// expected output: false
+
```

Comment:
Should we add one or two more with different types? I would like to see some strings in with numeric types. We shouldn't cover every case, though. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#examples

Reply:
In case I add 1 more example, size of the widget would change to big and it would take up most of the viewport like this:

![image](https://user-images.githubusercontent.com/100634371/201327974-cb30f3b8-2e4b-4b6f-aaa0-0b1a9fa65894.png)

Don't you think it's an overkill considering how simple function `isArray` is? Even `Array.reduce` uses standard size.

Reply:
Alright, thank you